ow_plexil/src/plans/TestFaultDependencies.plp

```diff
@@ -0,0 +1,43 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+#include "plan-interface.h"
+
+// Tests the FaultDependencies class. Depending on the XML file used, 
+// the fault info of a subsystem will be printed out.
+
+
+TestFaultDependencies:
+{
+
+  PrintNode:
+  {
+    Repeat true;
+    Wait 3;
+
+    if !Lookup(IsOperable("Power")){
+      log_info("Power Subsystem Inoperable.");
+      log_info("Power Subsystem Faulty: ", Lookup(IsFaulty("Power")));
+      log_info("Active Faults In Subsystem: ", Lookup(ActiveFaults("Power")), "\n");
+    }
+
+    if !Lookup(IsOperable("Antenna")){
+      log_info("Antenna Subsystem Inoperable.");
+      log_info("Antenna Subsystem Faulty: ", Lookup(IsFaulty("Antenna")));
+      log_info("Active Faults In Subsystem: ", Lookup(ActiveFaults("Antenna")), "\n");
+    }
```

Comment:
While performing Test 1, I could never get these statements to print. I was toggling combinations of `ant_pan_joint_lock_failure` and `ant_tilt_joint_lock_failure` on and off.

Reply:
The orange warning text does print.

Reply:
Because the ant_tilt_joint_lock_failure and the ant_pan_joint_lock_failure will not cause the entire subsystem to be inoperable this if statement will only trigger when the power is inoperable. 

Mike mentioned above that we dont have a way to make a subsystem inoperable with a combination of faults, which is a limitation in the current system. 

If we changed the XML to have either antenna fault impact the antenna subsytem, this if statement would trigger without the power faults.

Reply:
Interesting. So we would need the capability to specify a condition for operability, like `pan_inoperable && tilt_inoperable`. 

Understandable we lack this feature, that sounds like a tough problem. Thanks for clarifying.